### ISSUE TITLE
golangci-lint fails usually

### ISSUE THREAD

**Issue by JorTurFer** [kedacore/keda#2367]
### Proposal

There are several failures produced by timeouts in golangci-lint task:

- [https://github.com/kedacore/keda/runs/4359169018?check_suite_focus=true](https://github.com/kedacore/keda/runs/4359169018?check_suite_focus=true)
- [https://github.com/kedacore/keda/runs/4356029088?check_suite_focus=true](https://github.com/kedacore/keda/runs/4356029088?check_suite_focus=true)
- [https://github.com/kedacore/keda/runs/4354674889?check_suite_focus=true](https://github.com/kedacore/keda/runs/4354674889?check_suite_focus=true)

All of them with similar error:
```
Run golangci against the code...............................................Failed
- hook id: golangci-lint
- exit code: 4
```

Maybe the code base is big and we should increase the timeout to avoid the error.

**Comment by zroubalik** [comment]
Yeah, we should fix that. The current timeout is 5m though. I am gonna try to extend it to 10m.
